From 03dde14c2cc236e20456f24910dae4308dc96f16 Mon Sep 17 00:00:00 2001
From: Dan Lavu <dlavu@redhat.com>
Date: Fri, 5 Feb 2016 09:28:41 -0500
Subject: [PATCH 64/86] PAM: Fix man for pam_account_{expired,locked}_message

Reviewed-by: Jakub Hrozek <jhrozek@redhat.com>
(cherry picked from commit 09092b6535b711b9b734ed0c047c671de9e6cafd)
(cherry picked from commit ffe3df61d85c2890a31d627c5e6fe8890f1e26d8)
---
 src/man/sssd.conf.5.xml | 35 +++++++++++++++++++++++++++++------
 1 file changed, 29 insertions(+), 6 deletions(-)

diff --git a/src/man/sssd.conf.5.xml b/src/man/sssd.conf.5.xml
index cf2301f06d03b580f0bd5cea3567599af45eed02..d7e324f457a42403133c9d25f34d702d703fc9b8 100644
--- a/src/man/sssd.conf.5.xml
+++ b/src/man/sssd.conf.5.xml
@@ -1006,16 +1006,39 @@ fallback_homedir = /home/%u
                     <term>pam_account_expired_message (string)</term>
                     <listitem>
                         <para>
-                           If user is authenticating using SSH keys and
-                           account is expired then by default
-                           'Permission denied' is output. This output will
-                           be changed to content of this variable if it is
-                           set.
+                           Allows a custom expiration message to be set,
+                           replacing the default 'Permission denied'
+                           message.
+                        </para>
+                        <para>
+                            Note: Please be aware that message is only
+                            printed for the SSH service unless pam_verbostiy
+                            is set to 3 (show all messages and debug
+                            information).
+                        </para>
+                        <para>
+                            example:
+                            <programlisting>
+pam_account_expired_message = Account expired, please contact help desk.
+                            </programlisting>
+                        </para>
+                        <para>
+                            Default: none
+                        </para>
+                    </listitem>
+                </varlistentry>
+                <varlistentry>
+                    <term>pam_account_locked_message (string)</term>
+                    <listitem>
+                        <para>
+                           Allows a custom lockout message to be set,
+                           replacing the default 'Permission denied'
+                           message.
                         </para>
                         <para>
                             example:
                             <programlisting>
-pam_account_expired_message = Account expired, please call help desk.
+pam_account_locked_message = Account locked, please contact help desk.
                             </programlisting>
                         </para>
                         <para>
-- 
2.5.0

